identity-example/src/main/java/com/stripe/android/identity/example/ComposeExampleActivity.kt

```diff
@@ -0,0 +1,543 @@
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.github.kittinunf.result.Result
+import com.google.android.material.composethemeadapter.MdcTheme
+import com.stripe.android.identity.IdentityVerificationSheet
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+abstract class ComposeExampleActivity : ComponentActivity() {
+    protected abstract val getBrandLogoResId: Int
+
+    private val viewModel: IdentityExampleViewModel by viewModels()
+
+    data class IdentitySubmissionState(
+        val shouldUseNativeSdk: Boolean = true,
+        val allowDrivingLicense: Boolean = true,
+        val allowPassport: Boolean = true,
+        val allowId: Boolean = true,
+        val requireLiveCapture: Boolean = true,
+        val requireId: Boolean = false,
+        val requireSelfie: Boolean = false
+    )
+
+    sealed class LoadingState {
+        object Idle : LoadingState()
+        object Loading : LoadingState()
+        class Result(val vsId: String = "", val resultString: String) : LoadingState()
+    }
+
+    @Composable
+    fun ExampleScreen() {
+        val scaffoldState = rememberScaffoldState()
+        val coroutineScope = rememberCoroutineScope()
+        Scaffold(
+            scaffoldState = scaffoldState,
+            topBar = {
+                TopAppBar {
+                    Text(
+                        text = stringResource(id = R.string.compose_example),
+                        modifier = Modifier.padding(start = 10.dp),
+                        fontWeight = FontWeight.Bold,
+                        color = LocalContentColor.current
+                    )
+                }
+            }
+        ) {
+            Column(
+                modifier = Modifier
+                    .padding(it)
+                    .fillMaxWidth()
+                    .fillMaxHeight()
+            ) {
+                val (submissionState, onSubmissionStateChanged) = remember {
+                    mutableStateOf(IdentitySubmissionState())
+                }
+                NativeOrWeb(submissionState, onSubmissionStateChanged)
+                Divider()
+                AllowedDocumentTypes(submissionState, onSubmissionStateChanged)
+                RequireDocTypes(submissionState, onSubmissionStateChanged)
+                Divider()
+
+                val (loadingState, onLoadingStateChanged) = remember {
+                    mutableStateOf<LoadingState>(LoadingState.Idle)
+                }
+                var vsId by remember { mutableStateOf("") }
+                val configuration = remember {
```

Comment:
Alternatively, could create the configuration outside of Compose, since it doesn't use anything from Compose.

Reply:
done